ic10emu/src/interpreter.rs

```diff
@@ -46,6 +46,10 @@ pub enum ICError {
     StackIndexOutOfRange(f64),
     #[error("slot index out of range: '{0}'")]
     SlotIndexOutOfRange(f64),
+    #[error("pin index {0} out of range 0-6")]
```

Comment:
inconsistent error message, the other errors begins with a capital letter

(technically, errors should be lowercase, see [API guidelines](https://rust-lang.github.io/api-guidelines/interoperability.html?#examples-of-error-messages) and earlier section)